nebula/ui/components/MZComposerView.qml

```diff
@@ -128,7 +128,7 @@ ColumnLayout {
 
                     implicitHeight: MZTheme.theme.rowHeight
                     text: loader.composerBlock.text
-                    colorScheme: loader.composerBlock.style === MZComposerBlockButton.Primary ?  MZTheme.colors.normalButton : MZTheme.colors.destructiveButton
+                    buttonType: loader.composerBlock.style === MZComposerBlockButton.Primary ? "normal" : "destructive"
```

Comment:
Is that accessible as MZButtonBase.buttonNames.normal? Because i will make typos if i need to write those out.

Reply:
Updated this to be an enum, which it should have been from the beginning.